discourse-topic-check.sh

```diff
@@ -41,11 +29,11 @@ then
   baseBranch="$(curl -s $pr | jq -r '.base.ref')"
 fi
 
-for file in $(git diff --name-only "$baseBranch".. | grep -E "*\.rst" | grep -Fvxf $DOES_NOT_REQUIRE_DISCOURSE_TOPIC)
+for file in $(git diff --name-only "$baseBranch".. | grep "\.rst$" | grep -Fvxf $DOES_NOT_REQUIRE_DISCOURSE_TOPIC)
 do
-  fileToCheck=$file
-  if ! containsDiscourseTopic
+  if ! containsDiscourseTopic "$file"
   then
+    echo "${file} does not have a discourse topic"
```

Comment:
Comment for line 55/43: Nice to have,  but it would be nice if there were a hint about to resolve this using the GitHub action -- thinking of the Ash/external collaborator case where they submit a PR, it fails on CircleCI, and they don't know what to do besides excluding it.

Reply:
Curious, is there a reason why we wouldn't want to automatically run the GitHub Action?

Reply:
> Curious, is there a reason why we wouldn't want to automatically run the GitHub Action?

We probably only want to run this script when the content/structure of the RSTs has been finalized (directly pre-merge?), because the script uses information from the RSTs to create the topics.  Some of the information in a topic is not changeable once it's created (the `embed_url`), and we also want to avoid creating "invalid" topics (for appearances and because the titles and `embed_url`s are unique and will collide with attempts to create the "valid" topics).